configure.ac

```diff
@@ -97,6 +97,12 @@ AC_CONFIG_FILES([
 	contrib/pyzfs/Makefile
 	contrib/pyzfs/setup.py
 	contrib/zcp/Makefile
+	contrib/zstd/Makefile
+	contrib/zstd/zstd-1.4.4/Makefile
+	contrib/zstd/zstd-1.4.4/lib/Makefile
+	contrib/zstd/zstd-1.4.4/lib/common/Makefile
+	contrib/zstd/zstd-1.4.4/lib/compress/Makefile
+	contrib/zstd/zstd-1.4.4/lib/decompress/Makefile
```

Comment:
Something to think about:
The version of zstd is already kept track of. 
However its easier to update (and looks cleaner) we we just move `/lib` to `contrib/zstd/`

Reply:
yeah, I have thought about that,too, but I don't want to touch that know. remember our plan to make a static lib and then link it, so we can set the frame size? this is when we should move to `/lib/libzstd` anyways and can drop the versioning in folder names

Reply:
Okey, so it is planned to be fixed before final PR release, thats all thats important...